tests/storage/cdi_import/utils.py

```
@@ -0,0 +1,61 @@
+    Returns:
+        str: The node name where the pod is scheduled.
+    """
+    for sample in TimeoutSampler(
+        wait_timeout=TIMEOUT_1MIN,
+        sleep=TIMEOUT_5SEC,
+        func=lambda: importer_pod.instance.get("spec", {}).get(
+            "nodeName",
+        ),
+    ):
+        if sample:
+            return sample
```

Ruff (0.15.10)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)